tests/rptest/tests/debug_bundle_test.py

```diff
@@ -95,3 +95,7 @@ def test_post_debug_bundle(self, ignore_none: bool):
         res = admin.get_debug_bundle_file(filename=filename, node=node)
         assert res.status_code == requests.codes.ok, res.json()
         assert res.headers['Content-Type'] == 'application/zip', res.json()
+
+        res = admin.delete_debug_bundle_file(filename=filename, node=node)
+        assert res.status_code == requests.codes.no_content, res.json()
+        assert res.headers['Content-Type'] == 'application/json', res.json()
```

Comment:
chore/question: Can it be confirmed that the file was removed?

Reply:
Done